src/common.rs

```diff
@@ -16,16 +17,40 @@ impl Warning {
             line,
             check_name,
             message,
+            is_fixed: None,
         }
     }
+
+    pub fn line_number(&self) -> usize {
+        self.line.number
+    }
+
+    pub fn set_fixed(&mut self, val: bool) {
+        self.is_fixed = Some(val);
+    }
 }
 
 impl fmt::Display for Warning {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let fixed_prefix = match self.is_fixed {
+            Some(is_fixed) => {
+                if is_fixed {
+                    "  Fixed: "
+                } else {
+                    "Unfixed: "
```

Comment:
I wouldn't like to print this prefix before all warnings.
It would be great if we grouped warnings into 2 groups - Fixed and Unfixed.

Reply:
I will remove it, if we change the output format.

Reply:
I have changed the output format.